cli/README.md

```diff
@@ -4,7 +4,7 @@
 
 ### Requirements 
 
- * go 1.12
+ * go 1.12 and above 
 
 ### Build and Run 
```

Comment:
After 'make build' customer should be able to call  'cft' directly without the 'bin/' prefix ( or any other prefix) can we add it to the global path?

Reply:
I thought end user have to upload cli executable manually, not build it with GO. Even in case if we'll ask end user to build it he'll should rather do this with 

`go get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/cli` command, which will place built binary to $GOPATH/bin

Reply:
Thats fine, but that needs to be documented, how to use the CLI as a total new user.

Reply:
Agreed, I would propose to do documentation as last task, once all functionality will be implemented